rubix-spi/src/main/java/com/qubole/rubix/spi/fop/Poolable.java

```diff
@@ -66,6 +66,13 @@ public void setLastAccessTs(long lastAccessTs)
     this.lastAccessTs = lastAccessTs;
   }
 
+  public void destroy()
```

Comment:
nit: we may call it 'reset' or something.

Reply:
Also, as this. object won't be on queue anymore. Can't we get rid of this?

Reply:
This is here to catch any issues around reusing destroyed object in testing. destroy suits better as this object will not be reused again, reset implies this object is reusable.